iroha_config/iroha_config_derive/src/lib.rs

```diff
@@ -0,0 +1,346 @@
+#![allow(clippy::module_name_repetitions, missing_docs)]
```

Comment:
We shouldn't allow missing docs here to my mind :)

Reply:
Please add module level doc with description of the derive macro and corresponding attributes.